docs/state.md

```diff
@@ -236,7 +236,47 @@ AppStore
     'tabs.switch-to-new-tabs': boolean, // true if newly opened tabs should be focused immediately
     'tabs.tabs-per-page': number // number of tabs per tab page
   },
-  sites: {
+  // TODO sort alphabetically
```

Comment:
sorted collections is not easily possible with Immutable.JS right?

i'm trying to work around this by keeping separate sorted lists of siteKeys & scores ([redis-sorted-set](https://github.com/axelpale/redis-sorted-set)) in memory.